pr-statistics-spent-time.sh

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env bash
+
+
+# PARAMETERS ----------------------------
+GITHUB_REPO="status-im/status-dev-cli"
+DEBUG=1
+
+# PLATFORM ----------------------------
+case "$OSTYPE" in
+  darwin*)  OS="DARWIN" ;;
+  *)        OS="LINUX" ;;
+esac
+
+# FUNCTIONS ----------------------------
+
+github_pr() {
+  repo="$1"
```

Comment:
Can we print this? so if I run script with it says something like:

[Fetching PR stats from status-react]

Reply:
done in commit https://github.com/status-im/status-react/pull/2893/commits/0b287d778877586a7249552f78af30f2223027ac